### PR TITLE
Add SyedSohan.json

### PR DESCRIPTION
**Pull request by syedsohan**

### CODE REVIEW

File: public/data/SyedSohan.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+        "icon": "twitter"
+      },
+      {
+        "name": "Follow me on LinkedIn",
+        "url": "https://linkedin.com/in/syedsohan",
+        "icon": "linkedin"
+      }
+    ]
+  }
+
```
Comment (Panquesito7) [inline public/data/SyedSohan.json:23]:
```suggestion
}

```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

File: public/data/SyedSohan.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+      "url": "https://twitter.com/syedsohan27",
+      "icon": "twitter"
+    },
+    {
+      "name": "Follow me on LinkedIn",
+      "url": "https://linkedin.com/in/syedsohan",
+      "icon": "linkedin"
+    }
+  ]
+}
```
Comment (Panquesito7) [inline public/data/SyedSohan.json:22]: Still missing an endline.
```suggestion
}

```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Review by Panquesito7 — APPROVED** [review]

File: public/data/SyedSohan.json (hunk 2 above)
Comment (syedsohan) [reply public/data/SyedSohan.json]: Added.

File: public/data/SyedSohan.json [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+      "url": "https://github.com/syedsohan",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/syedsohan27",
+      "icon": "twitter"
+    },
+    {
+      "name": "Follow me on LinkedIn",
```
Comment (shreyaansjain06) [inline public/data/SyedSohan.json:17]:
```suggestion
      "name": "Connect with me on LinkedIn",
```

**Review by shreyaansjain06 — CHANGES_REQUESTED** [review]

**Review by Panquesito7 — APPROVED** [review]

**Review by shreyaansjain06 — APPROVED** [review]

File: public/data/SyedSohan.json (hunk 3 above)
Comment (Vyvy-vi) [reply public/data/SyedSohan.json]: This change is unneeded.
People can use whatever text they want.

File: public/data/SyedSohan.json [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+      "url": "https://twitter.com/syedsohan27",
+      "icon": "twitter"
+    },
+    {
+      "name": "Connect with me on LinkedIn",
+      "url": "https://www.linkedin.com/in/syedsohan",
+      "icon": "linkedin"
+    }
+  ]
+}
```
Comment (Vyvy-vi) [inline public/data/SyedSohan.json:22]:
```suggestion
}

```
Could a newline be added at EndOfFile?

**Review by Vyvy-vi — CHANGES_REQUESTED** [review]

File: public/data/SyedSohan.json (hunk 4 above)
Comment (syedsohan) [reply public/data/SyedSohan.json:22]: I can if you think it's really that much essential!

Comment (Vyvy-vi) [reply public/data/SyedSohan.json:22]: It isn't, but it's nice to keep files consistent 😅

Comment (syedsohan) [reply public/data/SyedSohan.json]: Added.

Comment (syedsohan) [reply public/data/SyedSohan.json]: @Vyvy-vi Added.

**Review by Panquesito7 — APPROVED** [review]

**Review by Vyvy-vi — APPROVED** [review]
lgtm!

File: public/data/SyedSohan.json (hunk 3 above)
Comment (adityaraute) [reply public/data/SyedSohan.json]: Follow is also valid actually.

**Pull request merged**